test/test_files/abl_godunov_hypre/abl_godunov_hypre.i

```diff
@@ -66,19 +66,46 @@ zhi.type =   "slip_wall"
 zhi.temperature_type = "fixed_gradient"
 zhi.temperature = 0.003 # tracer is used to specify potential temperature gradient
 
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#              HYPRE                    #
+#.......................................#
+
+mac_proj.verbose = 1
+mac_proj.mg_verbose = 1
+mac_proj.mg_cg_verbose = 1
+mac_proj.mg_rtol = 1.0e-8
+mac_proj.mg_atol = 1.0e-13
```

Comment:
For GPU runs, we might need this to be `1e-12`. You can look at CDash results and fine-tune this.

Reply:
atol or rtol?

Reply:
We have had to set `rtol = 1.0e-12` in other tests to get it to consistently pass tests on GPU.